src/utils/runConfigs.js

```diff
@@ -27,7 +27,14 @@ export const UniversalRun = {
   description: 'Run the flow on any agent with matching labels',
   ref: 'https://docs.prefect.io/api/latest/run_configs.html#universalrun',
   icon: 'fad fa-globe',
-  args: []
+  args: [
+    {
+      arg: 'env',
+      input_type: 'object',
+      label: 'Environment variables',
+      description: 'Additional environment variables to set in the container.'
+    }
+  ]
```

Comment:
I don't think this exists in the current version of `UniversalRun` but I might be wrong?

Reply:
You are correct! But it will exist soon https://github.com/PrefectHQ/prefect/pull/4383/files#diff-5f6b81fc3cefe99964972a1b88d0f0769a472d896c38b4f0e6a9a07cdb399d42R46.

This PR can't be merged until the core side is all set.

Reply:
Ahhhhh ok I thought I remembered that but wasn't sure if it was a dream 😅